lib/readGltf.js

```diff
@@ -10,14 +10,13 @@ var DeveloperError = Cesium.DeveloperError;
 
 module.exports = readGltf;
 
-function readGltf(gltfPath, callback) {
+function readGltf(gltfPath, options, callback, imageProcess) {
```

Comment:
I kind of unpacked the options further up, should I just be doing `options.xxx`?